### PR TITLE
Grammar fix for 15.0 blog post

### PR DESCRIPTION
**Pull request by Aweary**
Since "This" is the subject and singular, "have" should be "has".

### CODE REVIEW

**Comment by jimfb** [comment]
@gaearon How would you feel if we changed that whole sentence to:
 > React's use of the `<noscript>` tag has always been considered implementation details of how React targets the DOM.

Otherwise, it reads "these changes changes have always been considered implementation details".

**Comment by gaearon** [comment]
@jimfb Sounds good!

**Comment by Aweary** [comment]
@jimfb @gaearon updated my PR :+1:

File: docs/_posts/2016-04-07-react-v15.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ If you can’t use `npm` yet, we provide pre-built browser builds for your conve
...
+    We’ve also made use of these comment nodes to change what `null` renders to. Rendering to `null` was a feature we added in React 0.11 and was implemented by rendering `<noscript>` elements. By rendering to comment nodes now, there’s a chance some of your CSS will be targeting the wrong thing, specifically if you are making use of `:nth-child` selectors. React's use of the <noscript> tag has always been considered an implementation detail of how React targets the DOM. We believe they are safe changes to make without going through a release with warnings detailing the subtle differences as they are details that should not be depended upon. Additionally, we have seen that these changes have improved React performance for many typical applications.
```
Comment (gaearon) [inline docs/_posts/2016-04-07-react-v15.md]: Two small nits:

* Please put `<noscript>` in backticks (`) so it appears as code
* Please change React's to React’s (correct apostrophe that’s used throughout the post)

Thanks!

Comment (Aweary) [inline docs/_posts/2016-04-07-react-v15.md]: OS X always burning me with these apostrophes!

**Comment by Aweary** [comment]
@gaearon nits have been picked :+1:

**Comment by jimfb** [comment]
Woops, @zpao, thanks for catching that this needs to be merged to stable.

**Pull request merged**